velero-2-mixin/variables.libsonnet

```diff
@@ -0,0 +1,150 @@
+local g = import './g.libsonnet';
+local var = g.dashboard.variable;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+
+// Generates chained variables to use on on all dashboards
+{
+  new(this, varMetric):
+    {
+      local filteringSelector = this.config.filteringSelector,
+      local groupLabels = this.config.groupLabels,
+      local instanceLabels = this.config.instanceLabels,
+      local scheduleLabel = 'schedule',
+      local groupVarMetric = 'velero_backup_attempt_total',
+      local querySelectorVar = ['job', 'cluster', 'instance', 'schedule'],
```

Comment:
shouldn't those be a combination of groupLabels, instanceLabels?

Reply:
Yea, I've updated it now so its a combination of groupLabels, instanceLabels, and then the schedule label. Let me know what you think.